iceoryx_hoofs/include/iceoryx_hoofs/error_reporting/modules/hoofs/errors.hpp

```diff
@@ -0,0 +1,71 @@
+#ifndef IOX_HOOFS_ERROR_REPORTING_MODULES_HOOFS_ERRORS_HPP
+#define IOX_HOOFS_ERROR_REPORTING_MODULES_HOOFS_ERRORS_HPP
+
+#include "iceoryx_hoofs/error_reporting/error.hpp"
+
+namespace iox
+{
+namespace hoofs_errors
+{
+
+using error_code_t = iox::err::error_code_t;
+using module_id_t = iox::err::module_id_t;
+
+enum class ErrorCode : error_code_t
```

Comment:
With the suggestion from above we could have a `error_code_t::type` ... maybe we should drop the `_t` when `error_code_t` is a distinct type.

Reply:
You got me wrong. I did not suggest to replace the enum but with the suggestion to make `error_code_t` a distinct type instead of an alias the enum would not work anymore. The workaround would be to add a typedef to the `error_code_t` struct.